[quant][fx] Don't assume bias is a keyword-argument

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #71993
* #61646
* #61608
* __->__ #71426

dbr quantization makes faulty assumptions about which arguments are
passed as keyword arguments and which are passed as positional
arguments. This happens to work currently due to a quirk of how
`__torch_function__` is implemented in python functions, but will
break when the operators are moved to C++.

Differential Revision: [D33754262](https://our.internmc.facebook.com/intern/diff/D33754262)

peterbell10 committed:
[quant][fx] Don't assume bias is a keyword-argument

dbr quantization makes faulty assumptions about which arguments are
passed as keyword arguments and which are passed as positional
arguments. This happens to work currently due to a quirk of how
`__torch_function__` is implemented in python functions, but will
break when the operators are moved to C++.

[ghstack-poisoned]

peterbell10 committed:
Update on "[quant][fx] Don't assume bias is a keyword-argument"


dbr quantization makes faulty assumptions about which arguments are
passed as keyword arguments and which are passed as positional
arguments. This happens to work currently due to a quirk of how
`__torch_function__` is implemented in python functions, but will
break when the operators are moved to C++.

[ghstack-poisoned]

albanD commented:
@albanD has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33754262).

peterbell10 committed:
Update on "[quant][fx] Don't assume bias is a keyword-argument"


dbr quantization makes faulty assumptions about which arguments are
passed as keyword arguments and which are passed as positional
arguments. This happens to work currently due to a quirk of how
`__torch_function__` is implemented in python functions, but will
break when the operators are moved to C++.

Differential Revision: [D33754262](https://our.internmc.facebook.com/intern/diff/D33754262)

[ghstack-poisoned]

peterbell10 committed:
Update on "[quant][fx] Don't assume bias is a keyword-argument"


dbr quantization makes faulty assumptions about which arguments are
passed as keyword arguments and which are passed as positional
arguments. This happens to work currently due to a quirk of how
`__torch_function__` is implemented in python functions, but will
break when the operators are moved to C++.

Differential Revision: [D33754262](https://our.internmc.facebook.com/intern/diff/D33754262)

[ghstack-poisoned]

peterbell10 committed:
Update on "[quant][fx] Don't assume bias is a keyword-argument"


dbr quantization makes faulty assumptions about which arguments are
passed as keyword arguments and which are passed as positional
arguments. This happens to work currently due to a quirk of how
`__torch_function__` is implemented in python functions, but will
break when the operators are moved to C++.

Differential Revision: [D33754262](https://our.internmc.facebook.com/intern/diff/D33754262)

[ghstack-poisoned]

peterbell10 committed:
Update on "[quant][fx] Don't assume bias is a keyword-argument"


dbr quantization makes faulty assumptions about which arguments are
passed as keyword arguments and which are passed as positional
arguments. This happens to work currently due to a quirk of how
`__torch_function__` is implemented in python functions, but will
break when the operators are moved to C++.

Differential Revision: [D33754262](https://our.internmc.facebook.com/intern/diff/D33754262)

[ghstack-poisoned]

albanD reviewed (approved):
SGTM

peterbell10 committed:
Update on "[quant][fx] Don't assume bias is a keyword-argument"


dbr quantization makes faulty assumptions about which arguments are
passed as keyword arguments and which are passed as positional
arguments. This happens to work currently due to a quirk of how
`__torch_function__` is implemented in python functions, but will
break when the operators are moved to C++.

Differential Revision: [D33754262](https://our.internmc.facebook.com/intern/diff/D33754262)

[ghstack-poisoned]

albanD commented:
@albanD has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33754262).
